MyScript/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
The whole script differs greatly from the JS core tasks conception, so it should be moved to a different repository.